src/mem3.erl

```diff
@@ -254,6 +254,11 @@ allowed_nodes() ->
 nodes_in_zone(Nodes, Zone) ->
     [Node || Node <- Nodes, Zone == mem3:node_info(Node, <<"zone">>)].
 
+non_maintenance_nodes() ->
+    Nodes = [node() | erlang:nodes()],
+    {Modes, _} = rpc:multicall(config, get, ["couchdb", "maintenance_mode"]),
```

Comment:
@iilyak : ya that was my intent. Are you saying I shouldn't rely on this because the current node() should not be included in the check? Also, does order matter in this case?